packages/mdc-elevation/_mixins.scss

```diff
@@ -46,6 +47,113 @@
   }
 }
 
+///
+/// Called once per application to set up the global default elevation styles.
+///
+@mixin mdc-elevation-overlay-common($query: mdc-feature-all()) {
+  $feat-animation: mdc-feature-create-target($query, animation);
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-elevation-overlay-selector_ {
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-base-emit-once("mdc-elevation/common/structure") {
+        position: absolute;
+        top: 50%;
+        right: 50%;
+        bottom: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: inherit;
+        opacity: 0;
+        pointer-events: none;
+      }
+    }
+
+    @include mdc-feature-targets($feat-animation) {
+      @include mdc-base-emit-once("mdc-elevation/common/animation") {
+        transition: mdc-elevation-overlay-transition-value();
+      }
+    }
+  }
+
+  @include mdc-base-emit-once("mdc-elevation/common/color") {
+    @include mdc-elevation-overlay-fill-color($mdc-elevation-overlay-color, $query);
```

Comment:
It is a good practice to always name the optional sass parameters, so it doesn't break when we introduce new param to a mixin. `$query: $query`

Reply:
SGTM!